src/wp-includes/class-wp-theme-json.php

```diff
@@ -2735,6 +2735,8 @@ private static function get_block_nodes( $theme_json, $selectors = array(), $opt
 	 * @since 6.6.0 Setting a min-height of HTML when root styles have a background gradient or image.
 	 *              Updated general global styles specificity to 0-1-0.
 	 *              Fixed custom CSS output in block style variations.
+	 * @since 6.6.1 Avoid applying `:root :where()` wrapper to root and top-level element-only
```

Comment:
Should be part of `6.6.2`?

RC for `6.6.1` is already released https://make.wordpress.org/core/2024/07/18/wordpress-6-6-1-rc1-is-now-available/

Reply:
@mukeshpanchal27 I've mentioned the ticket to release leads, if it's bumped to 6.6.2 there will be plenty of time to update the docblock.